generator.go

```diff
@@ -378,6 +378,7 @@ func (g *generator) getContainers() ([]*RuntimeContainer, error) {
 			},
 			Name:         strings.TrimLeft(container.Name, "/"),
 			Hostname:     container.Config.Hostname,
+			HostnameFull: container.Config.Hostname + "." + container.Config.Domainname,
```

Comment:
Should this check that `Domanname` is not `""` before joining w/ `"."`?  If it's `""`, `HostnameFull` could get a value like `foo.`

Reply:
Yes, adding `Domainname` would be preferred.